<launch>
    <arg name="map_frame" default="icp_map"/>
    <arg name="control_odom_frame" default="odom"/>
    <arg name="rviz" default="true" />

    <node name="path_follower" pkg="aro_exploration" type="path_follower.py" output="screen">
        <rosparam subst_value="true">
            map_frame: $(arg map_frame)
            odom_frame: $(arg control_odom_frame)
            robot_frame: base_footprint
        </rosparam>

        Control parameters
        <rosparam file="$(dirname)/../../config/control.yaml" />

    </node>

    <include file="$(dirname)/aro_control_rviz.launch" pass_all_args="true" if="$(arg rviz)" />
</launch>
